.github/workflows/main.yml

```diff
@@ -13,9 +13,16 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0 # avoid shallow clone so nbgv can do its work.
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 5.0.100
     - uses: nuget/setup-nuget@v1.0.2
     - name: Restore
       run: nuget restore src/BehaviorsSdk.sln
-    - uses: microsoft/setup-msbuild@v1.0.1
-    - name: Build
+    - uses: microsoft/setup-msbuild@v1.0.2
+      with:
+        vs-version: '[16.8,16.9)'
+    - name: Build Solution
       run: msbuild src/BehaviorsSdk.sln /p:Configuration=Release
+    - name: Pack Behaviors
+      run: dotnet pack --configuration Release src/**/Microsoft.Xaml.Behaviors.csproj
```

Comment:
@punker76 Could you please check the created NuGet package and compare it with the official one to see if this packs the .design DLLs correctly?

@DVaughan Would this actually even be used for the upload to NuGet?

Reply:
@mgoertz-msft The dlls are there

![image](https://user-images.githubusercontent.com/658431/99779548-7c9e7880-2b15-11eb-9e6c-2d0f3059e18a.png)